.gitignore

```diff
@@ -80,7 +80,7 @@ typings/
 
 # Nuxt.js build / generate output
```

Comment:
Traces of Nuxt.js but I don't see vue anywhere in our project. Is this just from a pre-made .gitignore?

Reply:
It was, yes. I took it from a previous project when I 'included .gitignore' when creating a new repo in GitHub